lib/stat_tracker.rb

```diff
@@ -201,4 +288,64 @@ def most_tackles(season_id)
     team.team_name
   end
 
+
+
+
+  def season_data(all_seasons)
+    all_seasons = @games.map { |game| game[:season] }.uniq
+    @season_data = {}
+    season = []
+    @game_teams.select do |game_team|      
+      all_seasons.select do |season_id|
+        if season_id[0,4] == game_team.game_id[0,4]
```

Comment:
The index method here is interesting - you're using it to compare the game and team IDs in the two spreadsheets? I'd love to hear more about this approach.